apps/dh/api-dh/source/DataHub.WebApi.Tests/Snapshots/SchemaTests.ChangeTest.verified.graphql

```diff
@@ -9,22 +9,22 @@ interface Error {
 
 type Actor {
   id: UUID!
-  name: String!
-  glnOrEicNumber: String!
-  displayName: String!
-  marketRole: EicFunction
-  userRoles: [ActorUserRole!]!
-  status: ActorStatus!
-  gridAreas: [GridAreaDto!]!
-  contact: ActorContactDto
-  organization: Organization!
-  balanceResponsibleAgreements: [BalanceResponsibilityAgreement!]
-  credentials: ActorCredentialsDto
-  publicMail: ActorPublicMail
+  name: String! @cost(weight: "10")
+  glnOrEicNumber: String! @cost(weight: "10")
+  displayName: String! @cost(weight: "10")
+  marketRole: EicFunction @cost(weight: "10")
+  userRoles: [ActorUserRole!]! @cost(weight: "10")
+  status: ActorStatus! @cost(weight: "10")
+  gridAreas: [GridAreaDto!]! @cost(weight: "10")
+  contact: ActorContactDto @cost(weight: "10")
+  organization: Organization! @cost(weight: "10")
+  balanceResponsibleAgreements: [BalanceResponsibilityAgreement!] @cost(weight: "10")
+  credentials: ActorCredentialsDto @cost(weight: "10")
+  publicMail: ActorPublicMail @cost(weight: "10")
```

Comment:
The operation cost can be a cool feature, but I don't really think we need it at the moment. Maybe consider disabling it for now so that it doesn't clutter up the schema unneccesarily? Looking at the docs it seems like you can disable it like this:

```csharp
services
    .AddGraphQL()
    .ModifyRequestOptions(o =>
    {
        o.Complexity.Enable = false;
    });
```

Reply:
Have no luck disabling the feature. There does not seem to be an option in v14 that works; docs are useless.

Reply:
Worst setting name ever!